integration/tests/neon_evm/test_revision.py

```diff
@@ -1003,3 +1027,239 @@ def send_transaction_steps(holder_account, accounts):
         for acc in data_accounts:
             data_acc_revision_after = evm_loader.get_data_account_revision(acc)
             assert data_acc_revision_after == 3
+
+    def test_revision_revert_by_iterative_second_trx(
```

Comment:
let's add something about solana_call to the name ( the problem reproduced only for solana_calls, it is important)